hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/HoodieFileGroupReaderBasedParquetFileFormat.scala

```diff
@@ -359,4 +360,15 @@ class HoodieFileGroupReaderBasedParquetFileFormat(tableState: HoodieTableState,
   protected def getLogFilesFromSlice(fileSlice: FileSlice): List[HoodieLogFile] = {
     fileSlice.getLogFiles.sorted(HoodieLogFile.getLogFileComparator).iterator().asScala.toList
   }
+
+  protected def makeMappingIterator(iter: HoodieFileGroupReader.HoodieFileGroupReaderIterator[InternalRow],
+                                    f: Function[InternalRow, InternalRow]): Iterator[InternalRow] = {
```

Comment:
nit: better name for "f"?

Reply:
how about CloseableFileGroupRecordIterator  ?